pkg/R/RDD.R

```diff
@@ -46,7 +46,10 @@ setMethod("initialize", "PipelinedRDD", function(.Object, prev, func, jrdd_val)
   .Object@env$isCached <- FALSE
   .Object@env$isCheckpointed <- FALSE
   .Object@env$jrdd_val <- jrdd_val
+   # This tracks if jrdd_val is serialized
```

Comment:
Maybe also explain the purpose (since the pipelining logic is scattered around different places, it might become harder and harder to grasp as time goes by): "...used exactly once during the delayed computation of JRDD in `getJRDD`".